projects/vdk-plugins/vdk-jupyter/deploy-tutorial.md

```diff
@@ -0,0 +1,39 @@
+## Introduction
```

Comment:
``## Introduction`` 

That is not necessary. The template directly starts with the introduction with section title